IdentityCore/src/cache/metadata/accountMetadata/MSIDAccountMetadataCacheItem.m

```diff
@@ -82,12 +84,22 @@ - (BOOL)setCachedURL:(NSURL *)cachedURL
 
 - (NSURL *)cachedURL:(NSURL *)requestURL instanceAware:(BOOL)instanceAware
 {
+    if (self.isSignedOut) return nil;
```

Comment:
why we don't return cache URL if it's signed out? just curious :)

Reply:
My assumption is that, we are trying to block signed out account to be able to find FRT. Such a goal is achieved by checking the sign in state of an account before we do FRT query. 

Therefore, returning cache URL or not at the above line is not very important. I decided to be more protective to return nil cache URL if the account is signed out. What do you think? :)